### PR TITLE
[MIG] [18.0] import_export_templating: migrated module

### PR DESCRIPTION
**Pull request by ShwetaSerpentCS**
Responsible : Jayram

### CODE REVIEW

File: import_export_templating/wizard/wiz_download_template.py [hunk 1]
```diff
@@ -374,7 +274,10 @@ def import_data(self):
                             for x in sheet.row_values(rownum)
                         ]
                         model = self.ir_model.model
+                        print("\n\n model", model)
```
Comment (Trivedi-Vacha-SerpentCS) [inline import_export_templating/wizard/wiz_download_template.py]: Remove Print statements

**Review by Trivedi-Vacha-SerpentCS — CHANGES_REQUESTED** [review]
@ShwetaSerpentCS header-logo.png and index.html file requires your attention.
educate the developer too about the same.

File: import_export_templating/__manifest__.py [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 {
     "name": "Import / Export Templating",
-    "version": "17.0.1.0.0",
+    "version": "18.0.1.0.0",
     "license": "LGPL-3",
     "author": "Serpent Consulting Services Pvt. Ltd.",
     "maintainer": "Serpent Consulting Services Pvt. Ltd.",
```
Comment (Trivedi-Vacha-SerpentCS) [inline import_export_templating/__manifest__.py:8]: summary should not be blank.
also company website link should be https

**Comment by ketanserpentcs** [comment]
Testing is assigned to devraj

**Review by Trivedi-Vacha-SerpentCS — APPROVED** [review]

**Comment by Trivedi-Vacha-SerpentCS** [comment]
@ShwetaSerpentCS Icon should not be like this, the module is for data upload, not for any transportation(import/export).

**Comment by ketanserpentcs** [comment]
QA done with index by devraj

**Comment by Trivedi-Vacha-SerpentCS** [comment]
@ShwetaSerpentCS kindly verify index and banner

**Comment by ShwetaSerpentCS** [comment]
> @ShwetaSerpentCS kindly verify index and banner

Done, Ready to merge. @Trivedi-Vacha-SerpentCS

**Pull request merged**